app/_src/guides/progressively-rolling-in-strict-mtls.md

```diff
@@ -5,7 +5,7 @@ title: Progressively rolling in strict mTLS
 The [MeshTLS](/docs/{{ page.version }}/policies/meshtls/) policy allows you to gradually migrate services to mutual TLS without dropping a packet.
 
 ## Prerequisites
-- Completed [quickstart](/docs/{{ page.version }}/quickstart/kubernetes-demo/) to set up a zone control plane with demo application
+- Completed [quickstart](/docs/{{ page.version }}/quickstart/kubernetes-demo/) to set up a zone control plane with demo application and make sure you enabled mTLS from the quickstart.
```

Comment:
Wouldn't it be better to just add the step with configuring mTLS in this guide? I do this in my namespace scoped policies and I think that we need something specific we should just reconfigure it in guide to make sure users have right config